src/app/modules/players/players.view.html

```diff
@@ -2,36 +2,20 @@
 
   <div class="col-xs-12">
     <!-- TODO this string needs to be translated -->
-    <h2>seleziona giocatori</h2>
+    <h2>choose players</h2>
   </div>
 
   <div class="col-xs-12">
     <div ng-show="vm.savedPlayers.getAll().length">
       <!-- TODO this string needs to be translated -->
-      sono stati selezionati {{ vm.selectedPlayers.getAll().length }} giocatori
+      You have selected {{ vm.selectedPlayers.getAll().length }} player{{ vm.selectedPlayers.getAll().length != 1 ? "s" : "" }}
```

Comment:
Uno dei tanti errori della traduzione in questo modo è che non permette di gestire il seguente caso:
- ITA `[n] giocatori sono stati selezionati`
- EN `You have selected [n] players`

Come risolveresti questo problema?